docs/modules/usage/llms/azure-llms.md

```diff
@@ -1,55 +1,42 @@
 # Azure OpenAI LLM
 
-## Completion
-
 OpenHands uses LiteLLM for completion calls. You can find their documentation on Azure [here](https://docs.litellm.ai/docs/providers/azure).
 
-### Azure openai configs
+## Azure OpenAI Configuration
 
-When running the OpenHands Docker image, you'll need to set the following environment variables using `-e`:
+When running OpenHands, you'll need to set the following environment variable using `-e` in the
+[docker run command](/modules/usage/getting-started#installation):
 
 ```
-LLM_BASE_URL="<azure-api-base-url>"          # e.g. "https://openai-gpt-4-test-v-1.openai.azure.com/"
```

Comment:
Can we re-add something like this one, down the page, at line 25?

I _think_ it's a must, so it does need Advanced Options.

Reply:
Hmmm interesting...If you have to set base URL, would it be considered advanced options? And wouldn't it be easier to set through the UI than the command?

Reply:
Oh yes, I mean via the UI. I thought I saw you cannot add base URL except after you press Advanced Options. Do I misremember?

Reply:
I just checked: so if Advanced Options is not active, there's no text field for base URL. So the user needs to enable it, then enter base URL, and make sure "custom model" matches deployment name as they defined it in Azure.

Reply:
Ah got it. Because for Azure you always need a Base URL set right?

Reply:
@enyst whenever you have a few minutes, can you re-review this whole azure part? I rejigged it based on both your comments to always enable Advanced Options. Would really appreciate it.